### PR TITLE
Rectifying Package and Samples Version mismatch

### PR DESCRIPTION
**Pull request by sneakzttv**
version upgraded from 2.5.1 to 2.6 along with other packages.

### CODE REVIEW

**Review by rob1997 — APPROVED** [review]

**Comment by sneakzttv** [comment]
awaiting QA

File: Packages/io.chainsafe.web3-unity.hyperplay/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
     "name": "io.chainsafe.web3-unity.hyperplay",
-    "version": "1.0.0",
+    "version": "2.6.0",
```
Comment (rob1997) [inline Packages/io.chainsafe.web3-unity.hyperplay/package.json:3]: This should remain the same, I did not mention any version changes, just rectifying version mismatch between packages and samples.

File: Packages/io.chainsafe.web3-unity.marketplace/package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
     "name": "io.chainsafe.web3-unity.marketplace",
-    "version": "1.0.0",
+    "version": "2.6.0",
```
Comment (rob1997) [inline Packages/io.chainsafe.web3-unity.marketplace/package.json:3]: This should remain the same.

File: src/UnitySampleProject/Assets/Samples/web3.unity SDK HyperPlay/2.6.0.meta
Comment (rob1997) [inline src/UnitySampleProject/Assets/Samples/web3.unity SDK HyperPlay/2.6.0.meta:1]: This should remain the same.

**Review by rob1997 — CHANGES_REQUESTED** [review]

**Review by kantagara — APPROVED** [review]

File: Packages/io.chainsafe.web3-unity.hyperplay/package.json (hunk 1 above)
Comment (sneakzttv) [reply Packages/io.chainsafe.web3-unity.hyperplay/package.json:3]: image

?

Comment (sneakzttv) [reply Packages/io.chainsafe.web3-unity.hyperplay/package.json:3]: just tell me what versions you want each of the packages to be and ill amend the PR :)

Comment (sneakzttv) [reply Packages/io.chainsafe.web3-unity.hyperplay/package.json:3]: lets get real clear cut with this so we don't misunderstand each other

when i say sample folder version i mean these
image

when i say json package version i mean these
image

currently in this PR they are:

SDK
sample folder version: 2.6
package json version: 2.6

Hyperplay
sample folder version: 2.6
package json version: 2.6

Web3Auth
sample folder version: 2.6
package json version: 2.6

Lootboxes
sample folder version: 2.6
package json version: 2.6

Ramp
sample folder version: 2.6
package json version: 2.6

Marketplace
sample folder version: 2.6
package json version: 2.6

Please copy the above and replace the values to what you would like.
@kantagara can you weigh in on this too please.

Comment (rob1997) [reply Packages/io.chainsafe.web3-unity.hyperplay/package.json:3]: @sneakzttv all I requested was for version mismatch between samples and packages to be rectified, the easier way to do it is to delete samples and reimport samples, that is all. If it's still not clear I can take over, no problem

**Review by rob1997 — APPROVED** [review]

**Review by juans-chainsafe — APPROVED** [review]
These 2 errors appear in this PR
https://github.com/ChainSafe/web3.unity/issues/1033
https://github.com/ChainSafe/web3.unity/issues/1032

But they will be fixed in a different PR

**Pull request merged**